### ISSUE TITLE
instruction 0x7c, adci should add df?

### ISSUE THREAD

**Issue by sjaturner** [wd5gnr/1802UNO#2]
My datasheet has ADCI as:

    M(R(P)) + D + DF -> DF, D; R(P) + 1 -> R(P)

1802.cpp:261 seems to omit DF (noticed this comparing against my own implementation). I may be wrong though :-)

Just want to say that I'm very impressed with this project. Thank you!

**Comment by wd5gnr** [comment]
I just saw this. Will have to take a look.

**Comment by sjaturner** [comment]
I wrote a similar emulator (much less efficient that yours) and harnessed your code that. I fed random op codes into both until the results came out the same. The only things I found were the issue described in this thread and a weird corner case with short jumps at the end of a page. There's a description here: http://bitsavers.trailing-edge.com/components/rca/cosmac/MPM-201A_User_Manual_for_the_CDP1802_COSMAC_Microprocessor_1976.pdf

![image](https://user-images.githubusercontent.com/22588466/148999065-199e4e03-904f-4762-9cb9-403e2449e648.png)

I used my the emulator which I coded for this: https://github.com/sjaturner/microwriter_mw4

Thanks again for your excellent work!
